tap_hubspot/streams.py

```diff
@@ -36,11 +46,21 @@ class CallsStream(HubspotStream):
     path = "/crm/v3/objects/calls"
     primary_keys = ["id"]
 
+    def get_selected_properties(self) -> List[dict]:
+        selected = []
+        for key, value in self.metadata.items():
+            if(value.selected and len(key) > 0):
+                selected.append(key[-1])
+        
+        return selected
```

Comment:
Could we put this in `HubspotStream` so it isn't copy pasted in each subclass?

Also it could potentially be a list comprehension, something like

```python
return [
  key[-1] for key, value in self.metadata.items()
  if value.selected and len(key) > 0
]
```

my python is not amazing though so that may need changes

Reply:
I've moved it to HubspotStream in 76b49ccad97c1492f202afe1d11a8213fb52253b

Reply:
And done the list comprehension part in 403e01d81149be681fa9505bf8ab9d28255ba318

Reply:
Also tested locally the pipeline runs & correct properties are sent across 😇